templates/todo/projects/nodejs-mongo-aks/src/api/manifests/configprovider.tmpl.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: azconfig.io/v1
+kind: AzureAppConfigurationProvider
+metadata:
+  name: todo-api-config-provider
+spec:
+  endpoint: {{.Env.AZURE_APP_CONFIGURATION_ENDPOINT}}
+  target:
+    configMapName: todo-api-config
+  auth:
+    managedIdentityClientId: {{.Env.AZURE_AKS_IDENTITY_CLIENT_ID}}
```

Comment:
Is `{{.Env.AZURE_AKS_IDENTITY_CLIENT_ID}}` used for workload identity?

Reply:
No , it's not using workload identity